pkgs/applications/audio/puddletag/default.nix

```diff
@@ -24,6 +27,11 @@ python3Packages.buildPythonApplication rec {
 
   dontStrip = true;  # we are not generating any binaries
 
+  postFixup = ''
+    wrapPythonPrograms
+    wrapQtApp $out/bin/puddletag
```

Comment:
In general this change makes sense, though I don't know why some people say the package works for them without this change. You should also unmark it as broken and we'll see that your change is justified :).

In anyway, you should avoid double wrapping with:

```nix
  preFixup = ''
    makeWrapperArgs+=("''${qtWrapperArgs[@]}")
  ''
```

Instead of the command `wrapQtApp`. I'm also surprised that you need `wrapPythonPrograms` as well - it should be handled by the builder automatically..

Reply:
fixed. I consider myself a novice in writing nix expressions.

Reply:
The problem depends on your desktop. If it is qt and is a different version then the qt lib of puddletag, without this patch it will not run. If your system qt is the same version, no problem.